### ISSUE TITLE
"Helper/Guide" feature for snippets

### ISSUE THREAD

**Issue by calculuschild** [naturalcrit/homebrewery#3612]
### Your idea:

Some snippets are a bit complicated or unintuitive, and might benefit from some kind of "just-in-time" popup or a nearby link to a guide page. Mask images for example inject a couple lines of comments, and License snippets #3583 might benefit from more detailed FAQ.

I don't think it makes sense to bury links *inside* the snippet bar, but perhaps just a little popup icon when a snippet is injected, that can be clicked for more info. Or if we are really fancy, some kind of screenshot or animation gif thing showing the values being edited.

**Comment by 5e-Cleric** [comment]
I like this, but it is cluttering a lot of space, perhaps we can find some better arrangement.

**Comment by calculuschild** [comment]
Yeah I don't care specifically about the appearance. Just the general idea of some kind of help tool.

**Comment by ericscheid** [comment]
For situations where there are a lot of competing options on a sub-menu, we could have a help item stashed there...

<img width="539" alt="image" src="https://github.com/user-attachments/assets/f7d6b8d8-6b98-4228-ad31-05f65df07cc8">

Selecting that snippet menu item would present a modal dialog with content/links with more info.

**Comment by calculuschild** [comment]
> For situations where there are a lot of competing options on a sub-menu, we could have a help item stashed there...

Or perhaps a help item on the
parent menu item?

**Comment by 5e-Cleric** [comment]
> For situations where there are a lot of competing options on a sub-menu, we could have a help item stashed there...
> 
> <img alt="image" width="539" src="https://private-user-images.githubusercontent.com/2605467/379546690-f7d6b8d8-6b98-4228-ad31-05f65df07cc8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjk3ODIzMjUsIm5iZiI6MTcyOTc4MjAyNSwicGF0aCI6Ii8yNjA1NDY3LzM3OTU0NjY5MC1mN2Q2YjhkOC02Yjk4LTQyMjgtYWQzMS0wNWY2NWRmMDdjYzgucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTAyNCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEwMjRUMTUwMDI1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9OTg2NDc3NTM1NTA4MjhhZDBkNDIxMTYxMDcxODVmYmI3MWQ1ZDY1YzdhYzFhOGEzYjE3NmNlY2JhMjdmMGRlNCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.cG4Un2dYX0kn8vY4srhPhCI7haKE8tFlBcZMlXEnaXg">
> Selecting that snippet menu item would present a modal dialog with content/links with more info.

I think this option is horrible in terms of UI/UX.

There is no clear differentiation between snippet and help menu, at least give it a colored background
Also, it is more space used on screen, lots of it, while a simple `?` menu fits perfectly and doesn't clutter space.
